### PR TITLE
added content access permissions to install

### PR DESCRIPTION
**Pull request by boznik**
tried this on two jse and jsa builds and it worked. plz review.

### CODE REVIEW

File: stanford_private_page.install [hunk 1]
```diff
@@ -11,6 +11,18 @@ function stanford_private_page_install() {
...
+  $caspp = array();
+  $rids = array();
+  foreach($roles as $role_name) {
+    $role = user_role_load_by_name($role_name);
+    if(isset($role->rid)) {
+      $rids[] = $role->rid;
+    }
+  }
+  $caspp = array('view_own' => $rids, 'view' => $rids);
+  variable_set('content_access_stanford_private_page', $caspp);
```
Comment (sherakama) [inline stanford_private_page.install]: Add a variable get and merge the new values in with the existing values so we don't un-do anyone else that wants to do the same.

File: stanford_private_page.install [hunk 2]
```diff
@@ -11,6 +11,18 @@ function stanford_private_page_install() {
   // Let's keep the default settings defined in only one place
   $settings = stanford_private_page_get_settings();
   variable_set('stanford_stanford_private_page_settings', $settings);
+  // Setting access permissions for roles if they exist
+  $roles = array('administrator', 'site owner', 'editor', 'site member');
+  $caspp = array();
+  $rids = array();
+  foreach($roles as $role_name) {
+    $role = user_role_load_by_name($role_name);
+    if(isset($role->rid)) {
```
Comment (sherakama) [inline stanford_private_page.install]: Needs a space between if and opening (

File: stanford_private_page.install [hunk 3]
```diff
@@ -11,6 +11,18 @@ function stanford_private_page_install() {
   // Let's keep the default settings defined in only one place
   $settings = stanford_private_page_get_settings();
   variable_set('stanford_stanford_private_page_settings', $settings);
+  // Setting access permissions for roles if they exist
+  $roles = array('administrator', 'site owner', 'editor', 'site member');
+  $caspp = array();
+  $rids = array();
+  foreach($roles as $role_name) {
```
Comment (sherakama) [inline stanford_private_page.install]: Needs a space between foreach and opening (

File: stanford_private_page.install [hunk 4]
```diff
@@ -11,6 +11,18 @@ function stanford_private_page_install() {
   // Let's keep the default settings defined in only one place
   $settings = stanford_private_page_get_settings();
   variable_set('stanford_stanford_private_page_settings', $settings);
+  // Setting access permissions for roles if they exist
```
Comment (sherakama) [inline stanford_private_page.install]: missing period at end of comment.

**Comment by cjwest** [comment]
@boznik : I just rebuilt https://sites.stanford.edu/jse-dev/ and site member does not have permissions to view.

**Pull request merged**